l10n_ch_base_bank/invoice.py

```diff
@@ -19,6 +19,8 @@
 ##############################################################################
 from openerp.osv.orm import Model
 from openerp.tools import mod10r
+from openerp.osv import fields, osv
```

Comment:
./l10n_ch_base_bank/invoice.py:22:1: F401 'fields' imported but unused

Could you remove fields import, it fails on pep8 check on travis